packages/dawn4py/package.py

```diff
@@ -30,3 +31,6 @@ class Dawn4py(PythonPackage):
 
     # will test that dawn4py is importable after the install
     import_modules = ['dawn4py']
+
+    def setup_environment(self, spack_env, run_env):
```

Comment:
```suggestion
    def setup_build_environment(self, spack_env, run_env):
```
`setup_environment` has been deprecated and `setup_<run/build>_environment` should be used instead.